app/javascript/components/ArtistRequests.jsx

```diff
@@ -0,0 +1,226 @@
+import PropTypes from "prop-types";
+import React from "react";
+import CreateTransaction from "./CreateTransaction";
+import StyledModal from "./StyledModal";
+
+class ArtistRequests extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inbox: [],
+      display: [],
+      state: 0,
+      componentDidMount: false,
+    };
+    this.toggleState = ["Requests", "Pending", "Complete", "Archive"];
+  }
+
+  renderToggle() {
+    return this.toggleState.map((state, i) => {
+      return (
+        <div key={i} onClick={() => this.display(i)} className={(state === "Requests" ? "bg-charcoal" : "bg-mustard") + " pa2 mt2 toggle"}>
+          <h4 className="white">{state}</h4>
+        </div>
+      );
+    });
+  }
+
+  getRequestStatus = (request) => {
+    if (request.open) {
+      return "Pending";
+    } else {
+      if (request.receipt) {
+        return "Complete";
+      }
+      return "Closed";
```

Comment:
It looks like requests are initializing as "Closed"
![image](https://user-images.githubusercontent.com/43366398/48880420-781f1800-edb4-11e8-9e89-37837fd6aa53.png)